### ISSUE TITLE
Dashboard preview logo disappears after exiting out of telescope

### ISSUE THREAD

**Issue by antz22** [glepnir/dashboard-nvim#113]
Hi, after using dashboard-nvim for awhile (awesome plugin!), I noticed that if I went into 'find file' or another one of the commands on the dashboard and exited, the preview logo would disappear. I use telescope as the executive.

Here is a screen recording.

https://user-images.githubusercontent.com/59377837/140421498-9ea63f89-c60f-419a-b909-57489fd9d591.mp4

Here is my lua config.

```lua
vim.g.dashboard_footer_icon = '🐬 '

vim.g.dashboard_preview_command = 'cat'
vim.g.dashboard_preview_pipeline = 'lolcat'
vim.g.dashboard_preview_file = '~/.config/nvim/plug-config/neovim2.cat'
vim.g.dashboard_preview_file_height = 9
vim.g.dashboard_preview_file_width = 80

vim.g.dashboard_default_executive = 'telescope'

vim.g.dashboard_custom_section = {
    a = {description = {' Find File          '}, command = 'Telescope find_files find_command=rg,--files,--hidden,--ignore-file,~/.rgignore'},
    b = {description = {' Recently Used Files'}, command = 'Telescope oldfiles'},
    c = {description = {' Load Last Session  '}, command = 'SessionLoad'},
    d = {description = {' Find Word          '}, command = 'Telescope live_grep'},
    e = {description = {' Settings           '}, command = ':e ~/.config/nvim/nv-settings.lua'}
}
```

I might be missing something trivial but I do not know. Let me know if I have forgot to include any more info here. Thanks!

**Comment by pauldiloreto** [comment]
I can confirm I have the same issue. I am also using telescope as the executive & after closing it the dashboard header disappears.

I did notice that the dashboard works perfectly fine when set using `vim.g.dashboard_custom_header = { ... }`. This issue only occurs when we set it with the `vim.g.dashboard_preview_command` & `vim.g.dashboard_preview_file`. I tried both with and without `vim.g.dashboard_preview_pipeline = 'lolcat'`, and it still disappears after closing telescope.

Hope this helps. Thanks!
